spec/app_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative "../app"
+require "colorize"
+
+def reset_tables
+  seed_sql = File.read('spec/orders_items_seeds.sql')
+  connection = PG.connect({ host: ENV['HOST'], dbname: 'shop_test', user: 'postgres', password: ENV['PASSWORD'] })
+  connection.exec(seed_sql)
+end
+
+RSpec.describe Application do
+  before(:each) do
+    reset_tables
+  end
+
+  let(:io) {double :io}
+  let(:app) {Application.new('shop_test', io, ItemRepository.new, OrderRepository.new)}
+
+  it "Loops until valid option selected and then exits" do
+
+    expect(io).to receive(:puts).with("\n")
+    expect(io).to receive(:puts).with("Welcome to the shop management program!".colorize(:blue))
+    expect(io).to receive(:puts).with("\n")
+    expect(io).to receive(:puts).with("Please select one of the options:".colorize(:blue))
+    expect(io).to receive(:puts).with("1 = list all shop items")
+    expect(io).to receive(:puts).with("2 = create a new item")
+    expect(io).to receive(:puts).with("3 = list all orders")
+    expect(io).to receive(:puts).with("4 = create a new order")
+    expect(io).to receive(:puts).with("5 = exit program")
```

Comment:
As this code repeats, is there a private method you could add here to simplify the test code?